internal/pkg/agent/vault/seed.go

```diff
@@ -16,62 +17,116 @@ import (
 )
 
 const (
+	// seedFile is len(aesgcm.AES256) and contains only the random seed
+	// A default salt size of 8 is used with this seed file
 	seedFile = ".seed"
+	// seedFileV2 is len(aesgcm.AES256)+4 and contains the random seed followed by a non-zero salt size (little endian uint32)
+	seedFileV2     = ".seedV2"
+	seedFileV2Size = int(aesgcm.AES256) + 4
```

Comment:
nit: define a const of `int(aesgcm.AES256)` and derive seedFileV2Size from it + 4. Then use the AES256 const when reading the file in `getSeedV2` func